### ISSUE TITLE
Does Chalk support Chinese?

### ISSUE THREAD

**Issue by xjsXjtu** [nielsenramon/chalk#26]
Hi ,

I am new to Jekyll. I tried to install Chalk in my computer, but I got an error msg when runing 'bundle exec jekyll serve':

`$ bundle exec jekyll serve
Configuration file: C:/Users/xiongjs/github/xjsXjtu.github.io/_config.yml
Configuration file: C:/Users/xiongjs/github/xjsXjtu.github.io/_config.yml
            Source: C:/Users/xiongjs/github/xjsXjtu.github.io
       Destination: C:/Users/xiongjs/github/xjsXjtu.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
                    "\xA6" followed by "\n" on GBK
  Liquid Exception: "\xA6" followed by "\n" on GBK in /_layouts/post.html
jekyll 3.3.1 | Error:  "\xA6" followed by "\n" on GBK
`

GBK seems a Chinese standard. So I am wondering if Chalk supports Chinese.

Thanks a lot in advance.

**Comment by nielsenramon** [comment]
@xjsXjtu did you name your files in Chinese? I never worked with GBK so I have no idea tbh :)

**Comment by xjsXjtu** [comment]
No, I just clone and run the setup script. Then Jekyll serve .  Maybe there
was sth wrong with my computer configuration or Jekyll installation. I have
tried another Jekyll template, it seems work until now. Thanks a lot any
way:)

**Comment by leegaox** [comment]
I have the same problem!

**Comment by aboutyy** [comment]
I have the same problem but I do not name my file path in Chinese. It locates the post.html file. What are other options for the problem?

**Comment by nielsenramon** [comment]
@aboutyy can you share me your project please?

**Comment by aboutyy** [comment]
I just cloned Chalk on my PC and did not change anything of it.
My platform is Win10. The error information are similar to the poster here and when I changed post.html to another one that works fine for me in other project, it is ok and the default.html file presents a similar problem.
so I was wondering if it is the cross-platform or encoding problem?

**Comment by nielsenramon** [comment]
@aboutyy This could be a Windows thing. @leegaox @xjsXjtu did you guys also use Windows?

**Comment by xjsXjtu** [comment]
@nielsenramon ,yes. I am using Windows 7.

**Comment by leegaox** [comment]
I am using Windows 10

2017-02-10 22:17 GMT+08:00 xjs.xjtu <notifications@github.com>:

**Comment by nielsenramon** [comment]
@leegaox @xjsXjtu @aboutyy Can any of you try to remove this line: https://github.com/nielsenramon/chalk/blob/master/_includes/head.html#L2 and see if you still have the same issue please?

**Comment by aboutyy** [comment]
I removed https://github.com/nielsenramon/chalk/blob/master/_includes/head.html#L2 but I still got the same problem. @nielsenramon

**Comment by migueldemoura** [comment]
@aboutyy, can you try adding `encoding: utf-8` to /_config.yml to see if it helps?
If not, run `ruby -e 'puts Encoding::list.join("\n")'` and `ruby -v` and put here the output of both.

**Comment by lisanpeng** [comment]
I use Ubuntu and add utf-8 encoding to _config.yml. It doesn't work.
ruby version: 3.1p112 (2016-04-26) [x86_64-linux-gnu]
)'
ASCII-8BIT
UTF-8
US-ASCII
UTF-16BE
UTF-16LE
UTF-32BE
UTF-32LE
UTF-16
UTF-32
UTF8-MAC
EUC-JP
Windows-31J
Big5
Big5-HKSCS
Big5-UAO
CP949
Emacs-Mule
EUC-KR
EUC-TW
GB2312
GB18030
GBK
ISO-8859-1
ISO-8859-2
ISO-8859-3
ISO-8859-4
ISO-8859-5
ISO-8859-6
ISO-8859-7
ISO-8859-8
ISO-8859-9
ISO-8859-10
ISO-8859-11
ISO-8859-13
ISO-8859-14
ISO-8859-15
ISO-8859-16
KOI8-R
KOI8-U
Shift_JIS
Windows-1250
Windows-1251
Windows-1252
IBM437
IBM737
IBM775
CP850
IBM852
CP852
IBM855
CP855
IBM857
IBM860
IBM861
IBM862
IBM863
IBM864
IBM865
IBM866
IBM869
Windows-1258
GB1988
macCentEuro
macCroatian
macCyrillic
macGreek
macIceland
macRoman
macRomania
macThai
macTurkish
macUkraine
CP950
CP951
IBM037
stateless-ISO-2022-JP
eucJP-ms
CP51932
EUC-JIS-2004
GB12345
ISO-2022-JP
ISO-2022-JP-2
CP50220
CP50221
Windows-1256
Windows-1253
Windows-1255
Windows-1254
TIS-620
Windows-874
Windows-1257
MacJapanese
UTF-7
UTF8-DoCoMo
SJIS-DoCoMo
UTF8-KDDI
SJIS-KDDI
ISO-2022-JP-KDDI
stateless-ISO-2022-JP-KDDI
UTF8-SoftBank
SJIS-SoftBank

**Comment by migueldemoura** [comment]
@lisanpeng, can you provide more info? What version of Ubuntu, and is the error message and stack trace the same?

**Comment by lisanpeng** [comment]
Ubuntu version: 16.04.2 LTS.
Please see the error message below:
Configuration file: /home/lsp/Documents/chalk-master/_config.yml
            Source: /home/lsp/Documents/chalk-master
       Destination: /home/lsp/Documents/chalk-master/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
Error in _assets/stylesheets/light.scss:50 File to import not found or unreadable: ../yarn/fluidbox/src/css/_fluidbox.scss.
Load paths:
  /home/lsp/Documents/chalk-master/_assets/css
  /home/lsp/Documents/chalk-master/_assets/stylesheets
  /home/lsp/Documents/chalk-master/_assets/images
  /home/lsp/Documents/chalk-master/_assets/img
  /home/lsp/Documents/chalk-master/_assets/fonts
  /home/lsp/Documents/chalk-master/_assets/javascripts
  /home/lsp/Documents/chalk-master/_assets/js
  /home/lsp/Documents/chalk-master/_assets/yarn 
  Liquid Exception: File to import not found or unreadable: ../yarn/fluidbox/src/css/_fluidbox.scss. Load paths: /home/lsp/Documents/chalk-master/_assets/css /home/lsp/Documents/chalk-master/_assets/stylesheets /home/lsp/Documents/chalk-master/_assets/images /home/lsp/Documents/chalk-master/_assets/img /home/lsp/Documents/chalk-master/_assets/fonts /home/lsp/Documents/chalk-master/_assets/javascripts /home/lsp/Documents/chalk-master/_assets/js /home/lsp/Documents/chalk-master/_assets/yarn in /_layouts/post.html
jekyll 3.5.2 | Error:  File to import not found or unreadable: ../yarn/fluidbox/src/css/_fluidbox.scss.
Load paths:
  /home/lsp/Documents/chalk-master/_assets/css
  /home/lsp/Documents/chalk-master/_assets/stylesheets
  /home/lsp/Documents/chalk-master/_assets/images
  /home/lsp/Documents/chalk-master/_assets/img
  /home/lsp/Documents/chalk-master/_assets/fonts
  /home/lsp/Documents/chalk-master/_assets/javascripts
  /home/lsp/Documents/chalk-master/_assets/js
  /home/lsp/Documents/chalk-master/_assets/yarn

**Comment by migueldemoura** [comment]
Did you run `bin/setup`?
